provisioner_test.go

```diff
@@ -119,3 +123,75 @@ func TestNewDisembedderProvisioner(t *testing.T) {
 	assert.NoError(t, err)
 	testProvisionerSuccessful(t, *p, "linux", "amd64", DefaultVersionAstilectron, DefaultVersionElectron)
 }
+
+func TestZipShouldRemove(t *testing.T) {
+	l := log.New(log.Writer(), log.Prefix(), log.Flags())
+
+	appName := "Test"
+	dataDir, _ := filepath.Abs("./testdata/tmp")
+	vendorDir := filepath.Join(dataDir, "vendor") // https://github.com/CarsonSlovoka/go-astilectron/blob/e7796e5/paths.go#L56
```

Comment:
you shouldn't set all those variables manually below. instead you can use `a.Paths` methods once `a` has been created

Reply:
OK. https://github.com/asticode/go-astilectron/pull/358/commits/ae08b36c5a8f4b2b68add1fd5808ad5e8201c78a#diff-f5c6513a93e0d3a0d10fb30654a66c3ab4b7f05ed5b4dd9dd10ef379a14496adR139